app-admin/apg/apg-2.3.0b-r6.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Another Password Generator"
+HOMEPAGE="http://www.adel.nursat.kz/apg/"
+SRC_URI="http://www.adel.nursat.kz/apg/download/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+IUSE="cracklib"
+
+DEPEND="cracklib? ( sys-libs/cracklib )"
+
+src_prepare() {
+	default
+	if use cracklib; then
+		eapply "${FILESDIR}"/${P}-cracklib.patch
+		eapply "${FILESDIR}"/${PN}-glibc-2.4.patch
```

Comment:
I don't think you need to apply this conditionally.

Reply:
You're probably right, i've changed it to unconditionally apply the patch